vision6.module

```diff
@@ -608,36 +600,26 @@ function vision6_workbench_moderation_transition($node, $previous_state, $new_st
 function _vision6_node_beforesave($node) {
   global $base_url;
 
-  $wrapper = entity_metadata_wrapper('node', $node);
+  try {
```

Comment:
`entity_metadata_wrapper` requires using try/catch to avoid WSOF of unhandled exception.